task3/index.html

```diff
@@ -0,0 +1,174 @@
+                    <p class="comment">Комментарий 3</p>
+                    <span>&#9734;</span><span>&#9734;</span><span>&#9734;</span><span>&#9734;</span><span>&#9734;</span>
+                    <button class="buttonAddComment">Добавить комментарий</button>
+                </fieldset>
+            </td>
+             <td valign="top">
+               <fieldset>
+                    <img src="images/product.png" class="imgProduct" alt="product">   
+                    <p class="productName">Название</p>
+                    <p class="heshteg">#... #... #...</p>
+                    <p class="shortDescription">Краткое описание...</p>
+                    <p class="vendor">Вендор</p><a href="link.html">Ссылка</a>
+                    <div class="roundForSale"><p>%</p><p> До...</p></div>
+                    <p class="comment">Комментарий 1</p>
+                    <p class="comment">Комментарий 2</p>
+                    <p class="comment">Комментарий 3</p>
+                    <span>&#9734;</span><span>&#9734;</span><span>&#9734;</span><span>&#9734;</span><span>&#9734;</span>
+                    <button class="buttonAddComment">Добавить комментарий</button>
+                </fieldset>
+                <fieldset>
+                    <img src="images/product.png" class="imgProduct" alt="product">   
+                    <p class="productName">Название</p>
+                    <p class="heshteg">#... #... #...</p>
+                    <p class="shortDescription">Краткое описание...</p>
+                    <p class="vendor">Вендор</p><a href="link.html">Ссылка</a>
+                    <div class="roundForSale"><p>%</p><p> До...</p></div>
+                    <p class="comment">Комментарий 1</p>
+                    <p class="comment">Комментарий 2</p>
+                    <p class="comment">Комментарий 3</p>
+                    <span>&#9734;</span><span>&#9734;</span><span>&#9734;</span><span>&#9734;</span><span>&#9734;</span>
+                    <button class="buttonAddComment">Добавить комментарий</button>
+                </fieldset>
+                <fieldset>
+                    <img src="images/product.png" class="imgProduct" alt="product">   
+                    <p class="productName">Название</p>
+                    <p class="heshteg">#... #... #...</p>
+                    <p class="shortDescription">Краткое описание...</p>
+                    <p class="vendor">Вендор</p><a href="link.html">Ссылка</a>
+                    <div class="roundForSale"><p>%</p><p> До...</p></div>
+                    <p class="comment">Комментарий 1</p>
+                    <p class="comment">Комментарий 2</p>
+                    <p class="comment">Комментарий 3</p>
+                    <span>&#9734;</span><span>&#9734;</span><span>&#9734;</span><span>&#9734;</span><span>&#9734;</span>
+                    <button class="buttonAddComment">Добавить комментарий</button>
+                </fieldset>
+                <fieldset>
```

Comment:
In general, I'd advice you to separate content of each post into two parts: left and right. In this case it will be easier to style posts.

```
<article class="product>
     <div class="product-left-info">
          <img src="images/product.png" class="imgProduct" alt="product">
          <span>&#9734;</span><span>&#9734;</span><span>&#9734;</span><span>&#9734;</span><span>&#9734;</span>
     </div>
     <div class="product-right-info">
          <p class="product-name">Название</p>
            <p class="hashtags">#... #... #...</p>
            <p class="short-description">Краткое описание...</p>
            <p class="vendor">Вендор</p><a href="link.html">Ссылка</a>
            <div class="round-for-fale"><p>%</p><p> До...</p></div>
            <div class="comments">
                    <p class="comment">Комментарий 1</p>
                    <p class="comment">Комментарий 2</p>
                    <p class="comment">Комментарий 3</p>
            </div>
     </div>
</article>

````